### DISCUSSION TITLE
Is the anniversary edition support if we switch to classic graphics and sounds?

### DISCUSSION THREAD

**Discussion by Daryl-EndicottDavies** [LivingFray/HaloCEVR#16] [Q&A]
As per title, would like to support dev if possible.

**Comment by LivingFray** [comment]
I assume you're asking if you can use this mod on MCC if you play with the classic graphics?
Unfortunately its not that simple, these sorts of mods need to modify the code of the game and only work for the version(s) of the game the mods are written for. MCC and HaloPC have a lot of differences in their code that means not only does this mod not work with MCC but getting it to work would require a rewrite of the majority of the mod code

**Reply by Daryl-EndicottDavies** [reply]
Yes its part of MCC, thanks for the response!  I intend on loading up the mod on a G2 to take a look, assuming its ok with WMR?  Again, appreciate your work.
